src/web/timeline.rkt

```diff
@@ -270,6 +271,16 @@
               `(tr (td ,(~a count) "×")
                    (td (code ,(~a rule) " "))))))))
 
+(define (render-phase-problems problems)
+  `((dt "Problems")
+    (dd (table ([class "times"])
+          ,@(for/list ([rec (in-list (sort problems > #:key second))])
+              (match-define (list expr count) rec)
+              `(tr (td ,(~a count) "×")
+                   (td ,(if (string=? expr "#f")
```

Comment:
Ok, I understand why you're testing for the string `"#f"` here but c'mon this is pretty ugly. I think either:

1. This is a developer page, who cares, just show the `#f` for now; or
2. Don't stringify the expression if it is `#f`, you can do something like `(and k (~a k))`

Happy with either.

Also just use `equal?` instead of the type-specific `string=?`.